src/main/java/org/alfresco/utility/web/browser/WebBrowser.java

```diff
@@ -1254,4 +1256,37 @@ public List<String> getTextFromElementList(List<WebElement> elementsList)
     {
         return elementsList.stream().map(WebElement::getText).collect(Collectors.toList());
     }
+
+    public WebDriver.Navigation navigate() { return driver.navigate(); }
+
+    public WebDriver getDriver()
+    {
+        return driver;
+    }
+
+    public WebElement findElement(By by) {
+        return driver.findElement(by);
+    }
```

Comment:
I think we should use one style for brackets. In Alfresco we use the Allman Style.
You can use [alfresco-formatter.xml](https://github.com/Alfresco/alfresco-community-repo/blob/master/alfresco-formatter.xml).